source/features/pr-filters.tsx

```diff
@@ -64,25 +68,35 @@ async function addStatusFilter(): Promise<void> {
 		return;
 	}
 
-	// TODO: replace this with an API call
-	const hasCI = await fetchCIStatus();
+	const cachedCIStatus = cache.function(async () => {
+		// TODO: replace this with an API call
+		const statusIcon = await fetchCIStatus();
+		return statusIcon !== undefined;
+	}, {
+		expiration: 3,
+		cacheKey: () => __featureName__ + ':' + getRepoURL()
+	});
+
+	const hasCI = await cachedCIStatus();
 	if (!hasCI) {
 		return;
 	}
 
 	// Copy existing element and adapt its content
 	const statusFilter = reviewsFilter.cloneNode(true);
-	const dropdown = select('.select-menu-list', statusFilter)!;
+	statusFilter.id = '';
+
+	select('summary', statusFilter)!.firstChild!.textContent = 'Status '; // Only replace text node, keep caret
```

Comment:
I think `\u00A0` avoided line breaks in that place. Refer to the git blame

Reply:
Thanks, good catch!